package.json

```diff
@@ -79,16 +94,17 @@
   "devDependencies": {
     "@types/glob": "^8.1.0",
     "@types/mocha": "^10.0.1",
-    "@types/node": "20.2.5",
+    "@types/node": "^20.2.5",
     "@types/vscode": "^1.80.0",
     "@vscode/test-electron": "^2.3.2",
     "eslint": "^8.41.0",
     "glob": "^8.1.0",
     "mocha": "^10.2.0",
-    "typescript": "^5.1.3"
+    "typescript": "^5.4.2"
   },
   "dependencies": {
     "form-data": "^4.0.0",
-    "node-fetch": "^2.1.2"
+    "node-fetch": "^2.1.2",
+    "playcanvas": "^1.69.2"
```

Comment:
maybe worth adding a line to engine promotion doc to bump this number when we promote new engine to the editor